### PR TITLE
Quiver systems

### PR DESCRIPTION
**Pull request by Underewarrr**
Hello. Inspired @xydius solution I've created possibility to make use for quivers for both TFS
 (1.X based on otland/forgottenserver (https://github.com/otland/forgottenserver)
 and 0.X on Fir3element/3777 (https://github.com/Fir3element/3777)).

### CODE REVIEW

**Comment by MillhioreBT** [comment]
This still exists in TibiaRL today? if that's the case that's fine with me, although it's better to compare the itemId instead of using the name, I think it takes a few nanoseconds less to compare a number instead of a string, to any other item that they modify the name with Quiver, then it could cause some weird problem, maybe no know ;)

**Comment by joseluis2g** [comment]
Quiver is a new feature in Cipbia, like 12.61 feature.

File: src/item.cpp [hunk 1]
```diff
@@ -1238,7 +1238,7 @@ std::string Item::getDescription(const ItemType& it, int32_t lookDistance,
 		uint32_t volume = 0;
 		if (!item || !item->hasAttribute(ITEM_ATTRIBUTE_UNIQUEID)) {
 			if (it.isContainer()) {
-				volume = it.maxItems;
+				volume = item->getName() == "Quiver" ? 4 : it.maxItems;
```
Comment (kygov) [inline src/item.cpp:1241]: I think Item can be nullptr.

**Comment by nekiro** [comment]
Hardcoding names is never a good solution, this is very basic implementation, thus I wouldn't accept this code.

File: src/item.cpp (hunk 1 above)
Comment (Underewarrr) [reply src/item.cpp:1241]: This working fine. Just need to be attempted to slot system don't broke.

Comment (nekiro) [reply src/item.cpp:1241]: This code will crash the server when you look at the item in npc trade.

Comment (Underewarrr) [reply src/item.cpp:1241]: True.... I already start a new "new class" for "quivers"

Comment (Underewarrr) [reply src/item.cpp:1241]: Okay, already test this script and are very obsolete, to come useful to version TFS 1.3 + Ammo type Verify, and some another things are completely wrong.
If anyone going to use that keep in mind this is a VERY RAW SCRIPT for you guys have some knowledge base, this is not usefully script C++.
This is complete unstable and can be very dangerous as these people don't have knowledge about what they are doing on the source...

**Comment by EPuncker** [comment]
closing this based on the last comment of the PR author and also this can't be accepted in the current state

**Comment by Zbizu** [comment]
I would wait with this feature until we begi working on protocol 12

**Comment by Underewarrr** [comment]
@Zbizu  we have now !

**Comment by Underewarrr** [comment]
@nekiro 
#4105 This sould go to tests.

**Pull request closed without merging**